workers/generic-worker/gw-decision-task/tasks.yml

```diff
@@ -304,6 +304,11 @@ Mounts:
   git2.24.0.2:
     directory: git
     content:
+      linux:
+        amd64:
+          url: 'https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.24.1.tar.xz'
+          sha256: '723f24dce8fdd621a308b6187553fce7d5244205c065fe0a3aebd0b7c3f88562'
+          format: tar.xz
```

Comment:
Note, you'll need to update https://github.com/taskcluster/taskcluster/blob/6fa4d5d5f6cb5a0b5955777a7212e05ffd7927c5/workers/generic-worker/gw-decision-task/tasks.yml#L170 if you want this binary to get used by the task (rather than the system git installation).

Reply:
I thought this was how the code was tested, so I added some case of `tar.xz` to check. Isn't it the case ? IS this completely somethign else ?

Reply:
No this isn't a test itself, this is part of the task definition for the generic-worker CI tasks.